my-code/movie-search/src/Containers/MovieCatalogue/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { Link } from 'react-router-dom';
+
+import { fetchMovies } from '../../Actions';
+
+class MovieCatalogue extends Component {
+  renderCatalogue() {
```

Comment:
There's no need for `renderCatalogue`.
This class could be a functional component with the mapping of the movies directly in the render:

```jsx
const MovieCatalogue = ({ moviesList }) => (
  <div className="MovieCatalogue">
    <ul>
      {moviesList.Search.map(movie => (
        <li key={movie.imdbID} className="Card">
        <Link to={`/movies/${movie.imdbID}`}>
          <div className="Card-Overlay">
            <div className="Card-Info">
              <h4 className="Card-Title">{movie.Title}</h4>
              <span className="Card-Year">{movie.Year}</span>
            </div>
          </div>
          <img className="Card-Poster" src={movie.Poster} alt={movie.Title} />
        </Link>
      </li>
      ))}
    </ul>
  </div>
);
```

Reply:
I'll probably need you, Danilo or whoever to help me grasp the difference on this one. Thanks for pointing it out though!